### PR TITLE
Adding connector document MD file

### PR DESCRIPTION
**Pull request by cs-amol-phatangade**

### CODE REVIEW

File: docs/proofpoint-threat-response.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
+## About the connector
+Proofpoint Threat Response is a solution designed to help organizations manage and respond to cybersecurity threats. It provides tools and features to identify, investigate, and remediate security incidents.
+<p>This document provides information about the Proofpoint Threat Response Connector, which facilitates automated interactions, with a Proofpoint Threat Response server using FortiSOAR&trade; playbooks. Add the Proofpoint Threat Response Connector as a step in FortiSOAR&trade; playbooks and perform automated operations with Proofpoint Threat Response.</p>
+### Version information
```
Comment (cs-utkarsh-surana) [inline docs/proofpoint-threat-response.md:5]: Add a new line here

File: docs/proofpoint-threat-response.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+</td>
+</tr><tr><td>Username</td><td>Specify the Username used to access the Proofpoint Threat Response server to connect and perform automated operations.
+</td>
+</tr><tr><td>Password</td><td>Specify the Password used to access the Proofpoint Threat Response server to connect and perform automated operations.
+</td>
+</tr><tr><td>Verify SSL</td><td>Specifies whether the SSL certificate for the server is to be verified or not. <br/>By default, this option is set to True.</td></tr>
+</tbody></table>
+## Actions supported by the connector
+The following automated operations can be included in playbooks and you can also use the annotations to access operations:
+<table border=1><thead><tr><th>Function</th><th>Description</th><th>Annotation and Category</th></tr></thead><tbody><tr><td>Get List</td><td>Gets items for the specified list.</td><td>get_list <br/>Investigation</td></tr>
```
Comment (cs-utkarsh-surana) [inline docs/proofpoint-threat-response.md]: Change the Statement from Gets items for the specified list. to Gets items from the specified list.

File: proofpoint-threat-response/info.json [hunk 3]
```diff
@@ -489,8 +489,8 @@
           "name": "incident_id",
           "title": "Incident ID",
           "type": "text",
-          "description": "Specify the incident Id for add comment into.",
-          "tooltip": "Specify the incident Id for add comment into.",
+          "description": "Specify the incident Id.",
```
Comment (cs-utkarsh-surana) [inline proofpoint-threat-response/info.json]: Make Id to ID in all places

File: proofpoint-threat-response/info.json [hunk 4]
```diff
@@ -60,10 +60,10 @@
   "operations": [
     {
       "operation": "get_list",
-      "title": "Get List",
+      "title": "Get List of Indicators",
```
Comment (cs-utkarsh-surana) [inline proofpoint-threat-response/info.json]: The action name should go by Get Indicators List, This is the standard which we follow for all get actions make the corresponding changes in all such get actions.

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

**Review by cs-utkarsh-surana — APPROVED** [review]
LGTM

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**